doc/modules/model_evaluation.rst

```diff
@@ -1745,6 +1746,80 @@ the same does a lower Brier score loss always mean better calibration"
     and probability estimation." <https://drops.dagstuhl.de/opus/volltexte/2008/1382/>`_
     Dagstuhl Seminar Proceedings. Schloss Dagstuhl-Leibniz-Zentrum fr Informatik (2008).
 
+
+Calibration error
+-----------------
+
+The :func:`calibration_error` function computes the expected and maximum
+calibration errors as defined in [1]_ for binary classes.
```

Comment:
Would the general ECE for multiclass problems would be more desirable, as used in Guo 2017 (Ref 1)? This implementation seems targeted at binary classification. I believe you could use this as is for multiclass if you pass in `y_pred` as the maximum predicted probability and `y_true` as 1 if the predicted class equals the true class and 0 otherwise. But I would think it more desirable to just have the multiclass logic under the hood.